web-app/src/components/VerticalTabs.js

```diff
@@ -67,58 +68,75 @@ export default function VerticalTabs() {
     setValue(newValue);
   };
 
-  return (
-    <div className={classes.root}>
-      <Tabs
-        orientation="vertical"
-        value={value}
-        onChange={handleChange}
-        className={classes.tabs}
-      >
-        <Tab icon={<ImportContactsIcon />} label="Bibles" {...a11yProps(0)} />
-        <Tab icon={<CommentIcon />} label="Commentaries" {...a11yProps(1)} />
-        <Tab
-          icon={<FormatShapesIcon />}
-          label="Dictionaries"
-          {...a11yProps(2)}
-        />
-        <Tab icon={<ImageIcon />} label="Infographics" {...a11yProps(3)} />
-        <Tab icon={<VideocamIcon />} label="Videos" {...a11yProps(4)} />
-        <Tab
-          icon={<LocalLibraryIcon />}
-          label="Bible Stories"
-          {...a11yProps(5)}
-        />
-        <Tab icon={<EventIcon />} label="Reading Plans " {...a11yProps(6)} />
-      </Tabs>
-      <TabPanel value={value} index={0}>
-        <BibleContextProvider>
-          <h1>Bibles</h1>
-          <Bibles />
-        </BibleContextProvider>
-      </TabPanel>
-      <TabPanel value={value} index={1}>
-        <CommentaryProvider>
-          <Data />
-        </CommentaryProvider>
-      </TabPanel>
-      <TabPanel value={value} index={2}>
-        <DictonaryContextProvider>
-          <DictonaryData />
-        </DictonaryContextProvider>
-      </TabPanel>
-      <TabPanel value={value} index={3}>
-        Infographics
-      </TabPanel>
-      <TabPanel value={value} index={4}>
-        Videos
-      </TabPanel>
-      <TabPanel value={value} index={5}>
-        Bible Stories
-      </TabPanel>
-      <TabPanel value={value} index={6}>
-        Reading Plans
-      </TabPanel>
-    </div>
-  );
+	return (
+		<div className={classes.root}>
+			<Tabs
+				orientation="vertical"
+				value={value}
+				onChange={handleChange}
+				className={classes.tabs}
+			>
+				<Tab
+					icon={<ImportContactsIcon />}
+					label="Bibles"
+					{...a11yProps(0)}
+				/>
+				<Tab
+					icon={<CommentIcon />}
+					label="Commentaries"
+					{...a11yProps(1)}
+				/>
+				<Tab
+					icon={<FormatShapesIcon />}
+					label="Dictionaries"
+					{...a11yProps(2)}
+				/>
+				<Tab
+					icon={<ImageIcon />}
+					label="Infographics"
+					{...a11yProps(3)}
+				/>
+				<Tab icon={<VideocamIcon />} label="Videos" {...a11yProps(4)} />
+				<Tab
+					icon={<LocalLibraryIcon />}
+					label="Bible Stories"
+					{...a11yProps(5)}
+				/>
+				<Tab
+					icon={<EventIcon />}
+					label="Reading Plans "
+					{...a11yProps(6)}
+				/>
+			</Tabs>
+			<TabPanel value={value} index={0}>
+				<BibleContextProvider>
+					<h1>Bibles</h1>
+					<Bibles />
+				</BibleContextProvider>
+			</TabPanel>
+			<TabPanel value={value} index={1}>
+				<CommentaryProvider>
+					<Data />
+				</CommentaryProvider>
+			</TabPanel>
+			<TabPanel value={value} index={2}>
+				Dictionaries
```

Comment:
DictonaryContextProvider and DictionaryData can be added again , it got removed during the merge